### ISSUE TITLE
Not loading assets (image) when using non-default locale

### ISSUE THREAD

**Issue by MeelisH** [contentful/contentful.net#191]
Using` .NET Core 2.2` and latest `contentful.aspnetcore 4.3.0` package
In contentful I have following field, which is localized asset.

```
    {
      "id": "featuredImage",
      "name": "FeaturedImage",
      "type": "Link",
      "localized": true,
      "required": true,
      "validations": [],
      "disabled": false,
      "omitted": false,
      "linkType": "Asset"
    },
```

It's correctly fetched when using locale english. Fetch is done using the following code.

```
        public async Task<Post> GetPostBySlugLocale(string slug, string locale)
        {
            var queryBuilder = new QueryBuilder<Post>().LocaleIs(locale).FieldMatches(f => f.Slug, slug);
            var post = await _client.GetEntriesByType("post", queryBuilder);
```
On response image url is located in `post.featuredImage.file.url`.

When I use non-default locale `post.featuredImage.file.url` is null.
Altough when I check the response and go to `post.featuredImage.SystemProperties.Id` - the id is there and correct. But `post.featuredImage.file` is null. And by browsing response, locale seems to be correct too.

![image](https://user-images.githubusercontent.com/22073755/68209096-52961900-ffdb-11e9-975f-cc865a21e671.png)

**Comment by MeelisH** [comment]
To solve this, only thing that seems to be possible AOF is to make another round trip and fetch that asset by ID.

**Comment by Roblinde** [comment]
Hi @MeelisH 

If an asset is localized the file needs to be uploaded for every locale. If you look at the asset in the Contentful GUI does it have a file uploaded for the other locale? If you look at the response directly from the API, does the asset have a url there?

**Comment by MeelisH** [comment]
Hi @Roblinde 

Yes when I open the Asset it has only English uploaded, so that seems to be the problem . 

It's little-bit confusing, because in my "featuredImage" I have a localized Asset field for that asset, where I have set English and Finnish images, so you might think they are set and should be working. I didn't realize that this is not enaugh, and this asset itself needs to have uploaded images for different locales. Iam querying the "featuredImage" and this already has localized fields for this asset, and I am not quering this asset, which makes it confusing.

Closing
